### PR TITLE
graphql modified and next event logic

### PR DESCRIPTION
**Pull request by jtmst**
## Description
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/9576
Fixes bug where the fallback for featured events is not the next occuring event.

## Testing done
Tested with in a tugboat environment with data that simulates the conditions that results in the bug

## Screenshots

Screen Shot 2022-08-02 at 3 35 14 PM
Screen Shot 2022-08-02 at 3 34 57 PM
Screen Shot 2022-08-02 at 3 34 23 PM

## Acceptance criteria
- [ ] The top event from Upcoming should appear as the Featured Event on the VAMC page, if no events are Featured. 

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by maxx1128** [comment]
@jtmst Do you have a link to the live tugboat environment you used for your testing?

**Comment by jtmst** [comment]
> @jtmst Do you have a link to the live tugboat environment you used for your testing?

https://main-vpsyfavqes4gp7mmxayff3gvxqsntwzz.demo.cms.va.gov/

File: src/site/stages/build/drupal/graphql/healthCareRegionPage.graphql.js [hunk 1]
```diff
@@ -100,7 +100,7 @@ const nodeHealthCareRegionPage = `
     eventTeasersAll: reverseFieldOfficeNode(limit: 1000, filter: {conditions: [{field: "type", value: "event_listing"}]}) {
       entities {
         ... on NodeEventListing {
-          reverseFieldListingNode(sort: {field: "field_datetime_range_timezone", direction: ASC }, limit: 1, filter: {conditions: [{field: "type", value: "event"}, {field: "status", value: "1"}, { field: "field_datetime_range_timezone", value: [$today], operator: GREATER_THAN}]}) {
+          reverseFieldListingNode(sort: {field: "field_datetime_range_timezone", direction: ASC }, limit: 5, filter: {conditions: [{field: "type", value: "event"}, {field: "status", value: "1"}, { field: "field_datetime_range_timezone", value: [$today], operator: GREATER_THAN}]}) {
```
Comment (ryguyk) [inline src/site/stages/build/drupal/graphql/healthCareRegionPage.graphql.js:103]: Changing this to `limit: 5` gives us a much better chance that things won't go awry, but it doesn't actually guarantee that the defect doesn't emerge. Is there anything behind the decision to limit to five? Why not grab all?

**Review by ryguyk — COMMENTED** [review]
See my comments and let's discuss briefly.

**Review by JayDarnell — APPROVED** [review]
Nothing stands out as needing improvement. The code changes seem straight forward.

**Pull request merged**